### ISSUE TITLE
[a11y] /challenge-the-default/ page elements marked as aria-hidden are still focusable

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#15094]
## Description

There are several elements on the `/firefox/challenge-the-default/` page that are marked up using `aria-hidden="true"` but they are still focus-able DOM elements. A focus-able element with `aria-hidden="true"` is ignored as part of the reading order for screen readers, but still part of the focus order, making it’s state of visible or hidden unclear. 

https://dequeuniversity.com/rules/axe/4.10/aria-hidden-focus

### Steps to reproduce

1. Open https://www.mozilla.org/de/firefox/challenge-the-default/
2. Notice there are several element on the page using `aria-hidden` (such as the browser close/X button in the hero), but these are still possible to tab through to using a keyboard.

### Expected result

If these visible elements should be hidden from screen readers, then we should make them non-focus-able. Alternatively, if there are elements we could make more accessible then we could remove aria-hidden.

### Actual result

![image](https://github.com/user-attachments/assets/699acaaf-736d-4380-a45b-83c7305ca499)

### Environment

N/A

**Comment by alexgibson** [comment]
Duplicate of https://github.com/mozilla/bedrock/issues/15137
